src/utils/extractRepoName.ts

```diff
@@ -0,0 +1,18 @@
+function extractRepoName(issueUrl: string): string | null {
+    const regex = /https:\/\/github\.com\/([^/]+)\/([^/]+)\/issues\/(\d+)/;
+    const match = issueUrl.match(regex);
+
+    if (match && match.length === 4) {
+        const repoName = match[2]
+            .replace(/-/g, ' ')
+            .split(' ')
+            .map((word) => word.charAt(0) + word.slice(1))
+            .join(' ');
```

Comment:
Can you please explain what line 8-10 does?

Reply:
```js
            .split(' ')    // Split on spaces
            .map((word) => word.charAt(0) + word.slice(1))  // Capitalize first letter of each word
            .join(' '); // Join words back together
```

Reply:
Don't see capitalization part, is `toUpperCase()` missing?